inc/wp-components/classes/class-head.php

```diff
@@ -216,6 +218,33 @@ public function get_trailing_title() {
 		return ' | ' . get_bloginfo( 'name' );
 	}
 
+	/**
+	 * WordPress SEO Integration.
+	 */
+	public function set_wordpress_seo_meta() {
```

Comment:
We discussed this in Slack yesterday, but might be better or useful to make this more generic so that it can be used for other things. 

Maybe `set_additional_meta_tags()` or something like that?